pkg/github/teamreadwriter.go

```diff
@@ -383,3 +397,11 @@ func removeSubTeam(ctx context.Context, client *github.Client, orgID, teamID, su
 	}
 	return nil
 }
+
+func mapKeys[V any](m map[string]V) []string {
+	keys := make([]string, 0, len(m))
+	for k := range m {
+		keys = append(keys, k)
+	}
+	return keys
```

Comment:
It might be worth sorting the output so it's deterministic (`sort.Strings()`)

Reply:
good idea.